unity/src/test/scala/io/delta/unity/UCCatalogManagedClientSuite.scala

```diff
@@ -52,6 +54,40 @@ class UCCatalogManagedClientSuite extends AnyFunSuite with UCCatalogManagedTestU
     }
   }
 
+//  test("loadTable throws if version to load is greater than max ratified version") {
```

Comment:
why the commented out code?

Reply:
Hi Micah -- sorry, this PR was not yet ready to be reviewed